server/dbSchemas/attendanceSchema.js

```diff
@@ -14,7 +14,8 @@ const userSchema = new Schema({
 const User = mongoose.model('User', userSchema, 'user');
 
 const seatingLayoutSchema = new Schema({
-    name: { type: String, required: true },
+    name: { type: String, unique: true, required: true },
+    created_by: { type: ObjectId, ref: 'user', required: true},
```

Comment:
@JoaoPontes98 I believe there is no need for a `created_by` since Dawn mentioned that professors should be able to use any layouts (even if created by someone else). This is because the layouts will represent specific rooms, so if a teacher made the layout for the Dineen auditorium then any other teacher should be able to use it.
@ravenmills3 Is this correct?

Reply:
Yeah seating layouts can be accessed by other professors but having the created by field does not exclude it from being returned from a backend call. And that information can be useful for later

Reply:
Good point! Let's keep it, then